<launch>
  <arg name="ns" />
  <arg name="management_topic" default="sferes_fastsim" />
  <arg name="frequency" default="30"/>
  <arg name="creator_ns" default="unset"/>
  <arg name="creator_id" default="-1"/>
  <group ns="$(arg ns)">
    <node pkg="cafer_fastsim" type="cafer_fastsim" name="cafer_fastsim">
      <param name="settings" type="str" value="$(find cafer_fastsim)/envs/obst.xml"/>
      <param name="path" type="str" value="$(find cafer_fastsim)" />
      <param name="frequency" type="double" value="$(arg frequency)" />
      <param name="management_topic" type="string" value="$(arg management_topic)" />
      <param name="creator_ns" type="string" value="$(arg creator_ns)"/>
      <param name="creator_id" type="int" value="$(arg creator_id)"/>
      <param name="created_ns" type="string" value="$(arg ns)"/>
    </node>
  </group>
</launch>
